src/js/structs/Job.js

```diff
@@ -12,6 +12,13 @@ module.exports = class Job extends Item {
     return cmd;
   }
 
+  getCpus() {
+    // Default to the minimum number of cpus per Mesos offer
```

Comment:
I wonder if we should move this (and all other) default Mesos value to a constant file `MesosDefaults`. Out of scope, I guess.